packages/contracts/contracts/L1/AssetManager.sol

```diff
@@ -430,57 +379,35 @@ contract AssetManager is ISemver, IERC721Receiver, IAssetManager {
     /**
      * @inheritdoc IAssetManager
      */
-    function initUndelegateKgh(address validator, uint256 tokenId) external {
+    function undelegateKgh(address validator, uint256 tokenId) external {
         uint128 kroShares = _vaults[validator].kghDelegators[msg.sender].shares[tokenId].kro;
```

Comment:
Is it true that you just removed kgh share related things in undelegation functions, not corrected the functions to act correctly?

Reply:
+ I think it's better to remove and add new logics in the same PR in the future.

Reply:
I see.